Provis.Core/DTO/userDTO/UserConfirmEmailDTO.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Provis.Core.DTO.userDTO
+{
+    public class UserConfirmEmailDTO
+    {
+        public string ConfirmCode { get; set; }
```

Comment:
Consider fixing the names such userDTO by UpperCamelCase, ConfirmCode to ConfirmationCode etc.

Reply:
We created a TechDept ticket. Changes will be made during the next PR so that the changes do not affect the user history. https://app.clickup.com/t/1q4qun2